lint/rule_panel_datasource.go

```diff
@@ -8,6 +8,7 @@ func NewPanelDatasourceRule() *PanelRuleFunc {
 	return &PanelRuleFunc{
 		name:        "panel-datasource-rule",
 		description: "Checks that each panel uses the templated datasource.",
+		stability:   "stable",
```

Comment:
Having a const for this would be useful to avoid user error, and to have the compiler catch mismatched values.

See the [Severity const](https://github.com/grafana/dashboard-linter/blob/main/lint/lint.go#L9-L20).

Could probably be put in the `constants.go` file. - > https://github.com/grafana/dashboard-linter/blob/main/lint/constants.go

Reply:
I added const for stability.

Let me know if i misunderstood what you wanted :)